src/courseware/CoursewareContainer.jsx

```diff
@@ -310,6 +329,7 @@ class CoursewareContainer extends Component {
           nextSequenceHandler={this.handleNextSequenceClick}
           previousSequenceHandler={this.handlePreviousSequenceClick}
           unitNavigationHandler={this.handleUnitNavigationClick}
+          outlineNavigationSidebar={this.handleOutlineSidebarNavigationClick}
```

Comment:
```suggestion
          sidebarNavigationClickHandler={this.handleOutlineSidebarNavigationClick}
```
What about renaming this prop to follow the same convention as the other props?